YahooTickerDownloader.py

```diff
@@ -130,9 +132,12 @@ def main():
                 data.append(symbol.getRow())
             elif (symbol.exchange == args.Exchange):
                 data.append(symbol.getRow())
-
+                
         with open(downloader.type + '.csv', 'wb') as f:
-            f.write(data.csv.encode('UTF-8'))
+            if sys.version_info.major >= 3:
+                f.write(data.csv.decode('UTF-8'))
```

Comment:
This should have stayed 'encode()'. It now still does not work in python 3. :no_mouth: